zoo/example.txt

```diff
@@ -0,0 +1,24 @@
+Basic usage:
+
+Set up of a simple zoo with two cages for testing.
+
+from zoo.zoo import Zoo, Cage, BaseSpecies, Lion, Hyena, Wildebeest, Gazelle
+zoo = Zoo('Test Zoo')
+zoo.__repr__  # Displays the name of the zoo alongside the current number of cages and animals
+one_cage = zoo.add_cage('One Cage')  # Cages can be created directly from a Zoo object passing only their name.
```

Comment:
Nice, but I reckon it's a clearer API to instantiate cages rather than add cages as strings.  Example:

    one_cage = Cage('One Cage')
    zoo.add_cage(one_cage)

Another way to do it could be for the cages to know about the zoo, rather than vice versa.  That would look more like:

    zoo = Zoo('My zoo')
    cage = Cage('My cage', zoo=zoo)

Have a think about the strengths and weaknesses of each approach.